### ISSUE TITLE
Tinkerwell: cannot declare class error

### ISSUE THREAD

**Issue by jeffreyvr** [spatie/wordpress-ray#8]
Using [Tinkerwell](https://tinkerwell.app) (latest version, Version 2.11.0) while Ray is active, results in the following error (when executing code in Tinkerwell):

```
Fatal error: Cannot declare class Spatie\WordPressRay\Symfony\Component\Console\Output\Output, because the name is already in use in /Users/jeffreyvanrossum/Documents/Sites/wordpress/wp-content/plugins/wordpress-ray/vendor/symfony/console/Output/Output.php on line 30
``` 

So far I noticed when I uncomment the following bits from `autoload_static.php`, the error goes away and Ray appears to be working.

```php
'Symfony\\Component\\Console\\' =>
        array (
            0 => __DIR__ . '/..' . '/symfony/console',
        ),
```

Then the following error is introduced:

```
Fatal error: Cannot declare class Spatie\WordPressRay\Symfony\Component\VarDumper\Dumper\AbstractDumper, because the name is already in use in /Users/jeffreyvanrossum/Documents/Sites/wordpress/wp-content/plugins/wordpress-ray/vendor/symfony/var-dumper/Dumper/AbstractDumper.php on line 22
```

So when I uncomment this out as well, it is "resolved".

```php
'Symfony\\Component\\VarDumper\\' => 
        array (
            0 => __DIR__ . '/..' . '/symfony/var-dumper',
        ),
```

So I wonder if these classes are loaded twice somehow?

I'm not sure why this happens. I did notice this issue on the imposter-plugin repo:
https://github.com/TypistTech/imposter-plugin/issues/170

I tested this with the (at this time) current master branch of this repo, with no other active plugins.

Edit:

When I remove the imposter-plugin package, and then install the dependencies with composer, the issue is resolved. Not really a solution since all the namespace prefixes are lost, but it does indicate the issue is caused by the imposter-plugin.

Edit 2:

We can determine if Tinkerwell is used, but I would understand that is not desirable.

```php
$_SERVER['SCRIPT_NAME']; // would return tinkerwell
```

**Comment by freekmurze** [comment]
I consider this more an issue on Tinkerwell's part, as this package is working fine in WordPress.

Should it be easily fixable on our end, send a PR 👍

**Comment by jeffchown** [comment]
Hi @jeffreyvr - I've been trying to setup my dev environment to use the same tools (Ray, Tinkerwell, etc.) across all of my Laravel and Wordpress work and ran into the same error you described above when working on a Wordpress project.

As I want to be able to use Ray to help me debug my custom Wordpress plugins, I've installed Ray in my `/wp-content/mu-plugins` folder as instructed here https://spatie.be/docs/ray/v1/installation-in-your-project/wordpress#must-use-plugins

After some digging and a little experimentation, I found that, changing my `/wp-content/mu-plugins/ray-loader.php` file fixed the error and allowed me to use Ray and Tinkerwell together successfully with Wordpress :).

My new `/wp-content/mu-plugins/ray-loader.php` file contents are as follows:

```php
<?php

if (class_exists('LaravelTinkerwellDriver')) {
    if (! function_exists('ray')) {
        function ray() {}
    }
} else {
    require WPMU_PLUGIN_DIR.'/wordpress-ray/wp-ray.php';
}
```

Hope this helps anyone else who finds Ray and Tinkerwell as useful a combination as I do.
